ui/src/components/RepoSyncIcon.tsx

```diff
@@ -9,19 +9,40 @@ type RepoSyncIconPropsT = {
   className?: string
 }
 
+type WithTooltipPropsT = {
+  status: string
+  icon: JSX.Element
+}
+
+const WithTooltip: React.FC<WithTooltipPropsT> = ({ status, icon }: WithTooltipPropsT) => {
+  return (
+    <Tooltip content={status} placement='bottom'>
+      {icon}
+    </Tooltip>
+  )
+}
+
 export const RepoSyncIcon = ({ type, className = '' }: RepoSyncIconPropsT) => {
+  let icon
   switch (type) {
     case SYNC_STATUS.disabled:
-      return <CircleInformationFilledIcon className={cx('t-icon text-semantic-mutedIcon', { [className]: className !== '' })} />
+      icon = <CircleInformationFilledIcon className={cx('t-icon text-semantic-mutedIcon', { [className]: className !== '' })} />
+      break
     case SYNC_STATUS.succeeded:
-      return <CircleCheckFilledIcon className={cx('t-icon text-semantic-success', { [className]: className !== '' })} />
+      icon = <CircleCheckFilledIcon className={cx('t-icon text-semantic-success', { [className]: className !== '' })} />
+      break
     case SYNC_STATUS.failed:
-      return <CircleErrorFilledIcon className={cx('t-icon text-semantic-danger', { [className]: className !== '' })} />
+      icon = <CircleErrorFilledIcon className={cx('t-icon text-semantic-danger', { [className]: className !== '' })} />
+      break
     case SYNC_STATUS.queued:
-      return <ClockIcon className={cx('t-icon text-semantic-mutedIcon', { [className]: className !== '' })} />
+      icon = <ClockIcon className={cx('t-icon text-semantic-mutedIcon', { [className]: className !== '' })} />
+      break
     case SYNC_STATUS.running:
-      return <Spinner size="sm" className={className} />
+      icon = <Spinner size="sm" className={className} />
+      break
     default:
-      return <MinusIcon className={cx('t-icon text-semantic-mutedIcon', { [className]: className !== '' })} />
+      icon = <MinusIcon className={cx('t-icon text-semantic-mutedIcon', { [className]: className !== '' })} />
+      break
   }
+  return <WithTooltip status={`${type.charAt(0).toUpperCase()}${type.slice(1)}`} icon={icon} />
```

Comment:
This is cool, but maybe too fancy 😄 - I'd like to define slightly different tooltip text for each status type:

- `disabled`: `Sync is disabled`
- `succeeded`: `Last sync was successful`
- `failed`: `Last sync had an error`
- `queued`: `Sync is queued to run`
- `running`: `Sync is currently running`
- `default`: `No sync history`

Reply:
Done [62010c1](https://github.com/mergestat/fuse/pull/272/commits/62010c1e632c87cd9bde3666e777b70759af4dcf)